src/test/java/org/junit/tests/assertion/MultipleFailureExceptionTest.java

```diff
@@ -67,6 +68,16 @@ public void assertEmptyThrowsMutipleFailureExceptionForManyThrowables() throws E
         }
     }
 
+    @Test
+    public void assertEmptyErrorListConstructorFailure() throws Exception {
+        try {
+            new MultipleFailureException(Collections.<Throwable>emptyList());
+            fail();
```

Comment:
Sorry, haven't had my coffee yet :-)

My overall point is the code is easier to read if it's consistent and follows conventional idioms.

Alsi, if you only catch what you expect to be thrown, then if something else is thrown you get a useful stack trace in the failure